doc/users/whats_new.rst

```diff
@@ -24,6 +24,65 @@ revision, see the :ref:`github-stats`.
 New in Matplotlib 2.1
 =====================
 
+Enhancements to polar plot
+--------------------------
+
+The polar axes transforms have been greatly re-factored to allow for more
+customization of view limits and tick labelling. Additional options for view
+limits allow for creating an annulus, a sector, or some combination of the two.
+
+The :meth:`~matplotlib.axes.projections.polar.PolarAxes.set_rorigin` method may
+be used to provide an offset to the minimum plotting radius, producing an
+annulus.
+
+The :meth:`~matplotlib.projections.polar.PolarAxes.set_theta_zero_location` now
+has an optional :code:`offset` argument. This argument may be used to further
+specify the zero location based on the given anchor point.
+
+.. figure:: ../../gallery/pie_and_polar_charts/images/sphx_glr_polar_scatter_001.png
+   :target: ../../gallery/pie_and_polar_charts/polar_scatter.html
```

Comment:
Does this need to lose one set of ".."?

Reply:
Yes, waiting for link checking to finish so I can test locally and then push.